bootstrap/config/config.go

```diff
@@ -228,19 +228,16 @@ func (cp *Processor) loadCommonConfig(
 
 	// check that common config is loaded into the provider
 	// this need a separate config provider client here because the config ready variable is stored at the common config level
-	// TODO: refactor to add Client API that allows checking of a variable by passing an absolute path
-	commonConfigCheckClient, err := createProvider(cp.lc, common.CoreCommonConfigServiceKey, configStem, getAccessToken, configProviderInfo.ServiceConfig())
-	if err != nil {
-		return fmt.Errorf("failed to create provider client for common config check: %s", err.Error())
-	}
-	if err := cp.waitForCommonConfig(commonConfigCheckClient); err != nil {
-		return err
-	}
 	// load the all services section of the common config
 	commonConfigClient, err := createProvider(cp.lc, common.CoreCommonConfigServiceKey+allServicesKey, configStem, getAccessToken, configProviderInfo.ServiceConfig())
 	if err != nil {
 		return fmt.Errorf("failed to create provider for %s: %s", allServicesKey, err.Error())
 	}
+	// build the path for the common configuration ready value
+	commonConfigPath := fmt.Sprintf("%s/%s", configStem, common.CoreCommonConfigServiceKey)
```

Comment:
Why not also include `config.CommonConfigDone` here so it doesn't have to be appended later?

Reply:
ok. can do - was debating which way to go.